numpy/testing/_private/extbuild.py

```diff
@@ -182,35 +184,23 @@ def _make_source(name, init, body):
     return code
 
 
-def _c_compile(cfile, outputfilename, include_dirs=[], libraries=[],
-               library_dirs=[]):
+def _c_compile(cfile, outputfilename, include_dirs, libraries,
+               library_dirs):
+    link_extra = []
     if sys.platform == 'win32':
         compile_extra = ["/we4013"]
-        link_extra = ["/LIBPATH:" + os.path.join(sys.base_prefix, 'libs')]
+        link_extra.append('/DEBUG')  # generate .pdb file
```

Comment:
Not sure this will be robust, but I don't mind adding it if CI is happy

Reply:
This is just keeping the behavior we had before, there's another `sys.platform == 'win32'` block below I merged with this one.

I did delete the LIBPATH stuff because meson handles this now.